k8s/upgrades/0.9.0-1.0.0/README.md

```diff
@@ -0,0 +1,131 @@
+
+Before upgrading using helm, please review the default values available with latest stable/openebs chart. (https://raw.githubusercontent.com/helm/charts/master/stable/openebs/values.yaml).
+
+- If the default values seem appropriate, you can use the `helm upgrade --reset-values <release name> stable/openebs`.
+- If not, customize the values into your copy (say custom-values.yaml), by copying the content from above default yamls and edit the values to suite your environment. You can upgrade using your custom values using:
+`helm upgrade <release name> stable/openebs -f custom-values.yaml`
+
+#### Using customized operator YAML or helm chart.
+As a first step, you must update your custom helm chart or YAML with 1.0.0 release tags and changes made in the values/templates.
+
+You can use the following as references to know about the changes in 1.0.0:
+- openebs-charts [PR####](https://github.com/openebs/openebs/pull/2352) as reference.
+
+After updating the YAML or helm chart or helm chart values, you can use the above procedures to upgrade the OpenEBS Operator
+
+## Step 3: Upgrade the OpenEBS Pools and Volumes
+
+Even after the OpenEBS Operator has been upgraded to 1.0.0, the cStor Storage Pools and volumes (both jiva and cStor)  will continue to work with older versions. Use the following steps in the same order to upgrade cStor Pools and volumes.
+
+*Note: Upgrade functionality is still under active development. It is highly recommended to schedule a downtime for the application using the OpenEBS PV while performing this upgrade. Also, make sure you have taken a backup of the data before starting the below upgrade procedure.*
+
+Limitations:
+- this is a preliminary script only intended for using on volumes where data has been backed-up.
+- please have the following link handy in case the volume gets into read-only during upgrade
+  https://docs.openebs.io/docs/next/readonlyvolumes.html
+- automatic rollback option is not provided. To rollback, you need to update the controller, exporter and replica pod images to the previous version
+- in the process of running the below steps, if you run into issues, you can always reach us on slack
+
+
+### Upgrade the Jiva based OpenEBS PV
+
+Extract the PV name using `kubectl get pv`
+
+```
+NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS    CLAIM                          STORAGECLASS      REASON    AGE
+pvc-48fb36a2-947f-11e8-b1f3-42010a800004   5G         RWO            Delete           Bound     percona-test/demo-vol1-claim   openebs-percona             8m
+```
+
+```
+$ cd jiva
+$ ./jiva_volume_upgrade.sh pvc-48fb36a2-947f-11e8-b1f3-42010a800004
+```
+
+### Upgrade cStor Pools
+
+Extract the SPC name using `kubectl get spc`
+
+```sh
+NAME                AGE
+cstor-sparse-pool   24m
+```
+
+```sh
+$ cd cstor
+$ ./cstor_pool_upgrade.sh cstor-sparse-pool openebs
```

Comment:
let us give an example in getting the namespace of openebs.. and, let us not hardcode 'openebs' here. Mention `<namespace in which openebs is installed>`